mobile-app/app/components/InputHelperText.tsx

```diff
@@ -52,3 +52,41 @@ export function InputHelperText (props: InputHelperTextProps): JSX.Element {
     </ThemedView>
   )
 }
+export function InputHelperTextV2 (props: InputHelperTextProps): JSX.Element {
+  return (
+    <ThemedViewV2
+      light={tailwind('bg-transparent')}
```

Comment:
Can we just put this on style?

Reply:
yeap can

Reply:
nope, it will not work as intended on adroid